.ci/Jenkinsfile

```
@@ -185,7 +185,7 @@ def withKubernetes(Closure body) {
           chmod +x ${HOME}/bin/kubectl
           kubectl version --client
           ''') }
-        sh(label: "Create kind cluster", script: 'kind create cluster')
+        sh(label: "Create kind cluster", script: 'kind create cluster --config ../../kind-config.yaml')
```

If this config is in the root of the project, try with `BASE_DIR`

```suggestion
        sh(label: "Create kind cluster", script: 'kind create cluster --config ${BASE_DIR}/kind-config.yaml')
```

---

it seems to pass with `../../kind-config.yaml` 😅